src/main/resources/templates/earnings.html

```diff
@@ -0,0 +1,90 @@
+{# A test page for the dual pane page #}
+{% extends "base/li-sidebar-page" %}
+{% block title %} Exception Null - Earnings{% endblock %}
+
+{% block content %}
+<div class="earnings">
+	<div class="left">
+		<div class="card">
+			<div class="title">Earnings <span class="hint">this month, estimated</span></div>
+			<div class="big-number">$11.15</div>
+		</div>
+		<div class="card-spacing"></div>
+		<div class="card">
+			<div class="title">Reviews Completed <span class="hint">this month</span></div>
+			<div class="big-number">8</div>
+		</div>
+		<div class="card-spacing"></div>
+		<div class="card">
+			<div class="title">Rating <span class="hint">this month, average</span></div>
+			<div class="big-number">3.11</div>
+		</div>
+	</div>
+
+	<div class="right">
+		<div class="card">
+			<div class="title">History</div>
+			<canvas id="earning-review-history">8</canvas>
+		</div>
+	</div>
+</div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.bundle.min.js"></script>
+<script>
+	(function() {
+		window.onload = function() {
+			let ctx = document.getElementById('earning-review-history').getContext('2d');
+			let myChart = new Chart(ctx, {
+				type: 'bar',
+				data: {
+					labels: ["May (this month)", "April", "March", "February", "January"],
+					datasets: [{
+						type: 'bar',
+						label: 'Reviews',
+						data: [ 8, 4, 7, 8, 5 ],
+						backgroundColor: 'rgba(255, 99, 132, 0.2)',
+						yAxisID: 'y-axis-1',
+					}, {
+						type: 'line',
+						label: 'Earnings',
+						data: [null, '6.23', '10.11', '14.82', '7.56'],
+						pointBackgroundColor: 'rgba(153, 102, 255, 0.2)',
+						pointHoverBackgroundColor: 'rgba(153, 102, 255, 0.5)',
+						fill: false,
+						yAxisID: 'y-axis-2'
+					}]
+				},
+				options: {
+					responsive: true,
```

Comment:
That's a convenient way to make a responsive chart. Small thing, but it only scales up when the screen size increases; subsequent resizing of the screen doesn't scale down. Will leave it up to you whether or not it's big enough of an issue to warrant fixing for the demo.

![image](https://cloud.githubusercontent.com/assets/13812345/26491710/8e110a5c-41c5-11e7-9c0e-d6ed0bf4595c.png)

(increase screen size)

![image](https://cloud.githubusercontent.com/assets/13812345/26491726/9b4cf578-41c5-11e7-8dbc-8692fa06f3cd.png)

(decrease screen size)

![image](https://cloud.githubusercontent.com/assets/13812345/26491737/a495630e-41c5-11e7-8427-630e6a95b4d8.png)